pallene/builtins.lua

```diff
@@ -15,6 +15,7 @@ for lua_name, typ in pairs({
     ["string.char"]  = T.Function({T.Integer()}, {T.String()}),
     ["string.sub"]   = T.Function({T.String(), T.Integer(), T.Integer()}, {T.String()}),
     ["tofloat"]      = T.Function({T.Integer()}, {T.Float()}),
+    ["btype"]         = T.Function({T.Any()}, {T.String()}),
```

Comment:
Will it be possible to call this function "type" instead of "btype"?

Reply:
I would like it to be called `type` as well.

Reply:
There is no problem renaming the current `type` to `typealias` or even `alias`.

Reply:
Agreed. I would prefer renaming the current `type` to `typealias`.

Reply:
Changed to `typealias` in the new commit.